pkg/daemon/daemon.go

```diff
@@ -502,10 +574,19 @@ func (d *daemon) DeletePeriodicUpdate() {
 //  initPool check the guids that are already allocated by the running pods
 func (d *daemon) initPool() error {
 	log.Info().Msg("Initializing GUID pool.")
-	pods, err := d.kubeClient.GetPods(kapi.NamespaceAll)
-	if err != nil {
-		err = fmt.Errorf("failed to get pods from kubernetes: %v", err)
-		log.Err(err)
+
+	// Try to get pod list from k8s client in backoff loop
```

Comment:
i think for the initial startup of the daemon, we don't need to do exponential backoff WDYT ?

Reply:
Same reason as for subnet manager. Probably we can let ib-k8s gona fail here without backoff as on manager validation to notify user about configuration issues, but not sure how this will impact on UX.

Reply:
OK lets keep it as is